fili-core/src/test/groovy/com/yahoo/bard/webservice/filterbuilders/DruidInFilterBuilderSpec.groovy

```diff
@@ -0,0 +1,156 @@
+// Copyright 2018 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.filterbuilders
+
+import static com.yahoo.bard.webservice.druid.model.filter.Filter.DefaultFilterType.AND
+import static com.yahoo.bard.webservice.druid.model.filter.Filter.DefaultFilterType.IN
+import static com.yahoo.bard.webservice.druid.model.filter.Filter.DefaultFilterType.NOT
+
+import com.yahoo.bard.webservice.data.QueryBuildingTestingResources
+import com.yahoo.bard.webservice.data.filterbuilders.DruidFilterBuilder
+import com.yahoo.bard.webservice.data.filterbuilders.DruidInFilterBuilder
+import com.yahoo.bard.webservice.druid.model.filter.Filter
+import com.yahoo.bard.webservice.druid.model.filter.InFilter
+import com.yahoo.bard.webservice.druid.model.filter.NotFilter
+import com.yahoo.bard.webservice.web.ApiFilter
+
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class DruidInFilterBuilderSpec extends Specification {
```

Comment:
It looks like we're only testing single filter clauses at time. We really 
should have some tests with multiple filter clauses. If we had, the tests would
have caught the bug I pointed out above.